java-squid/src/main/java/org/sonar/java/se/NotMethodEqualsRelation.java

```diff
@@ -80,8 +81,33 @@ protected Boolean impliesNotMethodEquals() {
   }
 
   @Override
+  @CheckForNull
+  protected Boolean impliesGreaterThan() {
+    return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Null is returned but a "Boolean" is expected. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2447)